src/DeckGlMap/index.js

```diff
@@ -143,30 +190,18 @@ const DeckGlMap = ({
 	const onVectorLayerHover = (layerKey, featureKey, feature, x, y) => {
 		if (Tooltip) {
 			setTooltipData({
-				mapKey,
-				layerKey,
-				featureKey,
-				feature,
-				x,
-				y,
-			});
-		}
-	};
-	/**
-	 * @param layerKey {string}
-	 * @param info {Object}
-	 * @param event {Object}
-	 */
-	const onRasterLayerHover = (layerKey, info, event) => {
-		if (Tooltip) {
-			setTooltipData({
-				featureKey: true,
-				layerKey,
-				mapKey,
-				event,
-				info,
-				x: info.x,
-				y: info.y,
+				vector: [
+					{
+						mapKey,
+						layerKey,
+						featureKey,
+						feature,
+						x,
+						y,
+					},
+				],
+				raster: tooltipData.raster,
```

Comment:
why raster in vectorLayerHover?

Reply:
It use useState, so you must also reuse unmodified values.